charts/seed-bootstrap/prometheus-rules/recording-rules.rules.yaml

```diff
@@ -39,3 +69,48 @@ groups:
 
   - record: seed:container_network_transmit_bytes_total:sum_cp
     expr: sum(seed:container_network_transmit_bytes_total:sum_by_pod{namespace=~"((shoot-|shoot--)(\\w.+))"})
+
+  # Recording rules for the sum of all control-plane requests
+  - record: seed:kube_pod_container_resource_requests_cpu_cores:sum_cp
+    expr: sum(seed:kube_pod_container_resource_requests_cpu_cores:sum_by_pod{namespace=~"((shoot-|shoot--)(\\w.+))"})
+
+  - record: seed:kube_pod_container_resource_requests_memory_bytes:sum_cp
+    expr: sum(seed:kube_pod_container_resource_requests_memory_bytes:sum_by_pod{namespace=~"((shoot-|shoot--)(\\w.+))"})
+
+  # Recording rules for the sum of the entire seed vpa recommendations
+  - record: seed:vpa_status_recommendation:sum
+    expr: sum(seed:vpa_status_recommendation:sum_by_target{namespace=~"((shoot-|shoot--)(\\w.+))"})
+
+  # Recording rules for the sum of the entire seed hvpa applied recommendations
+  - record: seed:hvpa_status_applied_vpa_recommendation:sum
+    expr: sum(seed:hvpa_status_applied_vpa_recommendation:sum_by_target{namespace=~"((shoot-|shoot--)(\\w.+))"})
+
+  # Recording rules for node usage
+  - record: seed:node_cpu_seconds_total:sum_by_node
+    expr: sum(rate(node_cpu_seconds_total{mode!="idle"}[5m])) by (node)
+  - record: seed:node_memory_Active_bytes:sum_by_node
+    expr: sum(node_memory_Active_bytes) by (node)
+
+  # Recording rules for node allocatable
+  - record: seed:kube_node_status_allocatable_cpu_cores:max
+    expr: max(kube_node_status_allocatable_cpu_cores)
+  - record: seed:kube_node_status_allocatable_cpu_cores:sum
+    expr: sum(kube_node_status_allocatable_cpu_cores)
+  - record: seed:kube_node_status_allocatable_memory_bytes:max
+    expr: max(kube_node_status_allocatable_memory_bytes)
+  - record: seed:kube_node_status_allocatable_memory_bytes:sum
+    expr: sum(kube_node_status_allocatable_memory_bytes)
+  
+  # Recording rules for actual pods per node
+  - record: seed:kube_pod_info:sum_by_node
+    expr: count(kube_pod_info) by (node)
+
+  # Recording rules for allocatable pods per node
+  - record: seed:kube_node_status_allocatable_pods:max
+    expr: max(kube_node_status_allocatable_pods)
+  - record: seed:kube_node_status_allocatable_pods:sum
+    expr: sum(kube_node_status_allocatable_pods)
+
+  # Recording rules for active shoots
+  - record: seed:active_shoots_total:sum
```

Comment:
why do we need this?

Reply:
To be able to divide the sum_by_cp metrics by. This can give us avg requests, actuals per actual shoot (since the actual number of active shoots keep fluctuating).

Reply:
I don't think we need a recording rule for this. If you do need this you can use your expression or alternatively `count(shoot:availability)`.

Reply:
> I don't think we need a recording rule for this.

This is required to analyse the aggregate metrics as the active shoots keeps changing and we can not get per shoot stats without it.